app/views/layouts/_header.html.erb

```diff
@@ -20,17 +20,26 @@
           <a class="nav-link nav-icon nav-third" href="/"><%= image_tag "BW- CART ICON.svg", alt:"Cart" %><p>CHECK IN/OUT</p></a></li>
         <li class="nav-item active background-purple">
           <a class="nav-link nav-icon nav-fourth" href="/"><%= image_tag "BW - INVENTORY ICON.svg", alt:"View Inventory" %><p>CARRIERS</p></a></li>
-        <li class="nav-item dropdown">
-          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            Quick Access
-          </a>
-          <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-            <a class="dropdown-item" href="#">Action</a>
-            <a class="dropdown-item" href="#">Another action</a>
-            <div class="dropdown-divider"></div>
-            <a class="dropdown-item" href="#">Something else here</a>
-          </div>
-        </li>
+      </ul>
+      <ul class="navbar-nav ml-auto">
+        <% if user_signed_in? %>
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle d-flex align-items-center user-dropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              <span class="px-3 py-2 bg-info text-white font-weight-bolder rounded-circle"><%= "#{current_user.full_name[0]}".upcase %></span>
```

Comment:
Maybe it would be a nice idea to have `user_initial` method inside `Users` helper, and instead of using just first letter of the `full_name` we could go a bit more fancy, like this:
```ruby
module UsersHelper
  def user_initials(user)
    user.full_name.remove(/(\(|\[).*(\)|\])/).scan(/([[:word:]])[[:word:]]+/i).join
  end
end
```